probe-rs/targets/nRF53_Series.yaml

```diff
@@ -5,10 +5,16 @@ variants:
     cores:
       - name: application
         type: M33
-        arm_ap: 0
+        core_access_options:
+          Arm:
+            ap: 0
+            psel: 0
       - name: network
         type: M33
-        arm_ap: 1
+        core_access_options:
+          Arm:
+            ap: 1
+            psel: 0
```

Comment:
What's PSEL?

Reply:
Port select. @Yateki wanted that in too. It's for multi drop I think

Reply:
Ah, nice! it should be a u32 then.

Reply:
Ah! Changed it :)